canonical-web-and-design/jaas.ai
# Add the template for docs

## Done
Add docs templates with navigation highlighting.

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029/docs
- Check it looks good
- Check some pages are good and nav highlights
- Check /docs/commands page 

## Details
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/741
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/742
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/743
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/744
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/745

## Screenshots
![Screenshot_2019-07-19 Jujucharms Juju](https://user-images.githubusercontent.com/1413534/61520021-33ac5080-aa05-11e9-9e9d-c3cbf3ffe70f.png)
